src/pages/Login/Login.js

```diff
@@ -1,20 +1,85 @@
 import "./login.css";
 import { Navbar } from "../../components";
 import { Link } from "react-router-dom";
+import { useState } from "react";
+import axios from "axios";
+import { useAuth } from "../../Context/auth-context";
 
 const Login = () => {
+  const { authDispatch } = useAuth();
+
+  const [user, setUser] = useState({
+    email: "sudhanshu@gmail.com",
+    password: "sudhanshu",
+  });
+
+  const changeHandler = (e) => {
+    const name = e.target.name;
+    const value = e.target.value;
+    setUser({ ...user, [name]: value });
+  };
+
+  const loginHandler = async (e) => {
+    e.preventDefault();
+    try {
+      const response = await axios.post("/api/auth/login", {
+        email: `${user.email}`,
+        password: `${user.password}`,
```

Comment:
You can just pass the user.email value right.
No need to wrap this in template literal.